interfaces/nginx_route/v0/schemas/provider.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+class NginxRouteProviderSchema(BaseModel):
```

Comment:
it's sufficient to add:
```
class ProviderSchema(DataBagSchema):
    """Provider schema for nginx_route."""
```
to: /interfaces/nginx_route/v0/schema.py

Reply:
Updated, thanks!